ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/ANT/tasks/main.yml

```diff
@@ -64,12 +64,7 @@
   tags: ANT
 
 - name: Download ant-contrib
-  win_get_url:
-    url: https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.zip
-    dest: c:\temp\ant-contrib.zip
-    force: no
-    checksum: 22bae6c3ddf1a464b285784599eef8698f64dde24378c77e42522a536b88cbbc
-    checksum_algorithm: sha256
+  win_shell: c:\cygwin64\bin\curl -L -o /cygdrive/c/temp/ant-contrib.zip https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.zip
```

Comment:
We need to add in something here to verify the download on this download in the absence of the checksum check that was in `win_get_url`

Reply:
I couldn't consistently get the same shasum on that download URL (I have no idea why and not sure if we should be worried about this)

Reply:
I would be concerned.

Reply:
Agreed @karianna ... We had a SHA check with the previous `win_get_url` operation so if something is preventing that when using curl then that would definitely need to be understood before this can go in.

Reply:
(Since I'm going to be away for most of the next week, if these issues are resolved then feel free to dismiss my review)